pkgs/development/python-modules/numcodecs/default.nix

```diff
@@ -37,6 +38,10 @@ buildPythonPackage rec {
     msgpack
   ];
 
+  preBuild = if (stdenv.hostPlatform.isx86 && !stdenv.hostPlatform.avx2Support) then ''
```

Comment:
Nit: Is the isx86 check necessary?

Reply:
Yes, since `-mno-avx2` is only valid on i386 and x86_64.

Reply:
Got it, I see.